docker-compose.yml

```diff
@@ -103,4 +107,8 @@ services:
       - BB_PROMSTER_LEVEL=3
       - ETCD_URLS=http://etcd:2379
       - CLEAR_RR=true
-
+  
+  alertmanager:
+    image: labbsr0x/bb-promster-alertmanager
```

Comment:
@KarineValenca, please change `labbsr0x/bb-promster-alertmanager` to `prom/alertmanager`